### PR TITLE
Causely Integration Tile

### PR DESCRIPTION
**Pull request by esara**
### What does this PR do?

Causely - Datadog Integration Tile

### Motivation

Datadog provides a wide range of observability data and functionality including anomaly detection as Watchdog alerts. Causely builds on these signals and provides a Causal reasoning platform for Datadog customers.

### Review checklist

- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [X] Feature or bugfix has tests
- [X] Git history is clean
- [X] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [X] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: causely/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+### Improve Operational Efficiency
+- Reduce labor, data, and tooling costs by focusing on faster Mean Time to Repair (MTTR) and implementing efficient operational processes.
+### Accelerate The Delivery Of Features & Innovations
+- Enhance the speed and reliability of shipping new services and features to market, ensuring a competitive edge and meeting customer demands promptly.
+
+## Setup
+
+The Causely integration uses a [Datadog API key][2], which needs to be created by a Datadog admin.
+
+To connect Datadog with Causely, create monitors in Datadog. Refer to Datadog's [Alerting documentation][3].
```
Comment (estherk15) [inline causely/README.md:25]:
```suggestion
To connect Datadog with Causely, create monitors in Datadog. For more information, see the [Alerting documentation][3].
```

File: causely/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+See the [Causely documentation][4] to learn how to configure the Causely integration for Datadog.
```
Comment (estherk15) [inline causely/README.md:27]:
```suggestion
To learn how to configure the Causely integration for Datadog, see the [Causely documentation][4].
```

File: causely/manifest.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0ac3b5eb-46d2-4091-a001-e95dc5782609",
+  "app_id": "causely",
+  "display_on_public_website": false,
```
Comment (estherk15) [inline causely/manifest.json:5]: Is this intentionally false?

**Review by estherk15 — APPROVED** [review]
Minor formatting suggestions, if this is meant to public documentation the following should be true. Let me know if you have any questions!

> "display_on_public_website": true,

File: causely/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Our [Causal AI platform][1] automatically captures, represents and analyzes causality. We enable self-managed, resilient applications by bridging observability with automated orchestration.  Instead of constant noisy alerts, Causely tells you exactly what you need to know about all active and emerging application failures:
+
+- Why did it happen?
+- What was the root cause?
+- How can it be fixed?
+
+## What Causely offers:
+
+### Assure Continuous Application Reliability
```
Comment (emarsha94) [inline causely/README.md:14]: Within the overview, we avoid marketing material as this should be focused on: 
	1	What does your product do in 1-2 sentences
	2	What will customers get from this integration (the value of the integration & why it's helpful)
	3	What types of data sent and the value

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: causely/manifest.json (hunk 3 above)
Comment (emarsha94) [inline causely/manifest.json:5]:
```suggestion
  "display_on_public_website": true,
```

Comment (esara) [reply causely/manifest.json:5]: no, it was generated by ddev - changing it

File: causely/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# Causely - Assure Continuous Application Reliability using Causal AI
+
+## Overview
+
+In the fast-paced world of cloud-native development, ensuring application health and performance is critical. The application of Causal AI, with its ability to understand cause and effect relationships in complex distributed systems, offers the potential to streamline this process.
+
+A key enabler for this is application instrumentation that facilitates an understanding of application services and how they interact with one another. Datadog Application Performance Monitoring (APM) provides deep visibility into applications, enabling the identification of performance bottlenecks, troubleshoot issues, and optimize services.
```
Comment (emarsha94) [inline causely/README.md:7]:
```suggestion
```

File: causely/README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+# Causely - Assure Continuous Application Reliability using Causal AI
+
+## Overview
+
+In the fast-paced world of cloud-native development, ensuring application health and performance is critical. The application of Causal AI, with its ability to understand cause and effect relationships in complex distributed systems, offers the potential to streamline this process.
```
Comment (emarsha94) [inline causely/README.md:5]:
```suggestion
In the fast-paced world of cloud-native development, ensuring application health and performance is critical. The application of Causal AI, with its ability to understand cause and effect relationships in complex distributed systems, offers the potential to streamline this process.
```

This sounds like marketing copy. Could you update this to remove the marketing copy and provide 1-2 sentences on what your product does?

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: causely/README.md (hunk 6 above)
Comment (esara) [reply causely/README.md:5]: removed

Comment (esara) [reply causely/README.md]: removed

File: causely/README.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+The [Causal AI platform][1] automatically captures, represents and analyzes causality. It enables self-managed, resilient applications by bridging observability with automated orchestration. 
+
+Instead of flood of alerts, Causely tells you exactly what you need to know about all active and emerging application failures. With Causely, understand why a failure happened, what the root cause was, and how to fix it.
+
+Use Causely to:
+
+- **Assure continuous application reliability**: Guarantee that KPIs, SLAs, SLOs, and SLIs are consistently met to maintain application reliability and performance.
+- **Improve operational efficiency**: Reduce labor, data, and tooling costs by focusing on faster Mean Time to Repair (MTTR) and implementing efficient operational processes.
+- **Accelerate the delivery of features and innovations**: Enhance the speed and reliability of shipping new services and features to market, ensuring a competitive edge and meeting customer demands promptly.
```
Comment (emarsha94) [inline causely/README.md:13]: The overview is missing what the Causely <> Datadog integration does. Are lines 11-13 the specific data the integration is pulling from Datadog?

File: causely/README.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+## Setup
+
+The Causely integration uses a [Datadog API key][2], which needs to be created by a Datadog admin.
```
Comment (emarsha94) [inline causely/README.md:17]: Can you provide the exact steps a user will take? What happens with the API key once created?

File: causely/manifest.json [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+      }
+    ],
+    "classifier_tags": [
+      "Category::Containers",
+      "Category::Kubernetes",
+      "Category::Incidents",
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```
Comment (emarsha94) [inline causely/manifest.json:32]: Add relevant data types from this [list](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags)
ex: 

 "Queried Data Type::<FILL IN>"

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: causely/manifest.json (hunk 9 above)
Comment (esara) [reply causely/manifest.json:32]: done

File: causely/README.md (hunk 8 above)
Comment (esara) [reply causely/README.md:17]: the exact config is in the referenced documentation, but let me expand on what Causely will after it is configured

File: causely/README.md (hunk 7 above)
Comment (esara) [reply causely/README.md:13]: yes, that is the goal with the information gathered from datadog

File: causely/README.md (hunk 4 above)
Comment (esara) [reply causely/README.md]: see below

File: causely/CHANGELOG.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - Causely
+
+## 1.0.0 / 2024-03-26
```
Comment (emarsha94) [inline causely/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2024-04-16
```

**Review by emarsha94 — APPROVED** [review]

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Please add a logo in .svg format

**Comment by esara** [comment]
> Please add a logo in .svg format

done, thanks

File: causely/README.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+To connect Datadog with Causely, create monitors in Datadog. For more information, see the [Alerting documentation][3].
```
Comment (emarsha94) [inline causely/README.md:19]: This can be removed or moved under the paragraph below

Comment (esara) [reply causely/README.md]: done, thanks

**Review by emarsha94 — APPROVED** [review]

**Review by james-eichelbaum — APPROVED** [review]

File: causely/README.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+The [Causal AI platform][1] automatically captures, represents and analyzes causality. It enables self-managed, resilient applications by bridging observability with automated orchestration. 
+
+Instead of flood of alerts, Causely tells you exactly what you need to know about all active and emerging application failures. With Causely, understand why a failure happened, what the root cause was, and how to fix it.
+
+Use Causely to:
+
+- **Assure continuous application reliability**: Guarantee that KPIs, SLAs, SLOs, and SLIs are consistently met to maintain application reliability and performance.
+- **Improve operational efficiency**: Reduce labor, data, and tooling costs by focusing on faster Mean Time to Repair (MTTR) and implementing efficient operational processes.
+- **Accelerate the delivery of features and innovations**: Enhance the speed and reliability of shipping new services and features to market, ensuring a competitive edge and meeting customer demands promptly.
```
Comment (emarsha94) [inline causely/README.md:13]: Can you please include what data the integration is specifically sending to data and the value?

Comment (esara) [reply causely/README.md:13]: yes, of course, done

Comment (emarsha94) [reply causely/README.md:13]: This should be added to the overview. I am not seeing where you mention what data is sent into Datadog .Could you make sure to clarify which data is pulled out and what data is sent in?

Comment (esara) [reply causely/README.md:13]: I added an additional sentence to the overview, please let me know if this is sufficient description at this section.

File: causely/manifest.json
Comment (emarsha94) [inline causely/manifest.json:1]: @james-eichelbaum do you know what this needs to pass the validation? I think it's '['

File: causely/manifest.json [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+    "dashboards": {
+      "Causely Dashboard": "assets/dashboards/causely_dashboard.json"
+    },
+    "integration": {
+      "source_type_name": "Causely",
+      "configuration": {},
+      "events": {
+        "creates_events": true
+      }
+    },
```
Comment (emarsha94) [inline causely/manifest.json:47]:
```suggestion
    }
```

Comment (esara) [reply causely/manifest.json]: sorry for the mistake, thank you for finding it!

File: causely/manifest.json [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+    "classifier_tags": [
+      "Category::Containers",
+      "Category::Kubernetes",
+      "Category::Incidents",
+      "Offering::Integration",
+      "Queried Data Type::Metrics",
+      "Queried Data Type::Events",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```
Comment (emarsha94) [inline causely/manifest.json:34]: please update this to include submitted data type: https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags

File: causely/manifest.json [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Causal AI platform automatically captures, represents and analyzes causality.",
+    "title": "Causely",
+    "media": [
+      {
+        "media_type": "image",
```
Comment (emarsha94) [inline causely/manifest.json:15]: Can you add an image of data in Datadog?

File: causely/CHANGELOG.md [hunk 16]
```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - Causely
+
+## 1.0.0 / 2024-04-16
```
Comment (emarsha94) [inline causely/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2024-06-20
```

Comment (esara) [reply causely/CHANGELOG.md]: done, thank you

File: causely/manifest.json (hunk 15 above)
Comment (esara) [reply causely/manifest.json:15]: added a screenshot of the dashboard with an event, thank you!

File: causely/manifest.json (hunk 14 above)
Comment (esara) [reply causely/manifest.json]: added events, thanks!

File: causely/README.md [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+
+Causely integrates with Datadog to provide automated root cause analysis for Datadog Watchdog events. Causely uses the Datadog API key to fetch the configured monitors and use the triggered Watchdog events as input to the Causal AI reasoning platform and identifies the root cause of the anomalies and alerts. Causely sends back the remediated root case as events to be viewed in the Causely Datadog dashboard.
```
Comment (emarsha94) [inline causely/README.md:15]:
```suggestion
Causely integrates with Datadog to provide automated root cause analysis for Datadog Watchdog events. Causely uses the Datadog API key to fetch the configured monitors and use the triggered Watchdog events as input to the Causal AI reasoning platform and identifies the root cause of the anomalies and alerts. Causely sends back the remediated root case as events to Datadog which can be viewed in the Causely Datadog dashboard.
```

Comment (emarsha94) [reply causely/README.md:15]: Should it be root case or root cause?

Comment (esara) [reply causely/README.md]: than you, fixed

File: causely/images/Causely_Remediated_RootCauses.png
[the remark this answers is not in the document]
Comment (esara) [reply causely/images/Causely_Remediated_RootCauses.png:1]: done, thanks!

File: causely/assets/dashboards/causely_dashboard.json
[the remark this answers is not in the document]
Comment (esara) [reply causely/assets/dashboards/causely_dashboard.json:1]: I added a log and a description, I hope it helps

File: causely/assets/dashboards/causely_dashboard.json [hunk 18]
```diff
@@ -0,0 +1,113 @@
+{
```
Comment (emarsha94) [inline causely/assets/dashboards/causely_dashboard.json:1]: Sending you a dashboard template via email that you can use and fill out so it aligns with out best practices. Please make sure you fill out what is included in the template.

**Comment by emarsha94** [comment]
👋 Quick heads up that I will be out of office next week so there may be a delay in the review from my end until July 8th. With that said, other teams will continue to review the PR. Thanks in advance for your patience!

**Review by DrkSephy — APPROVED** [review]
Dashboard changes looks good semantically, but please resolve all of @emarsha94's comments before merging 🙏

File: causely/assets/dashboards/causely_dashboard.json (hunk 18 above)
Comment (esara) [reply causely/assets/dashboards/causely_dashboard.json:1]: thanks, I tried to follow the template and reworked the previous dashboard components, I hope it is ok

**Review by emarsha94 — APPROVED** [review]

File: causely/images/Causely_Remediated_RootCauses.png
Comment (emarsha94) [inline causely/images/Causely_Remediated_RootCauses.png:1]: please update the image to reflect the new dashboard

**Review by emarsha94 — CHANGES_REQUESTED** [review]

**Comment by emarsha94** [comment]
Hi @esara Could you add back the service check file? That looks to be why the validate repository check is failing

File: causely/images/Causely_Remediated_RootCauses.png
Comment (esara) [reply causely/images/Causely_Remediated_RootCauses.png:1]: thank you, that helped!

File: causely/CHANGELOG.md [hunk 19]
```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - Causely
+
+## 1.0.0 / 2024-06-20
```
Comment (emarsha94) [inline causely/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2024-08-01
```

**Review by emarsha94 — APPROVED** [review]

File: causely/CHANGELOG.md (hunk 19 above)
Comment (esara) [reply causely/CHANGELOG.md]: done, thanks!

File: causely/CHANGELOG.md [hunk 20]
```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - Causely
+
+## 1.0.0 / 2024-86-01
```
Comment (emarsha94) [inline causely/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2024-07-31
```

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: causely/CHANGELOG.md (hunk 20 above)
Comment (esara) [reply causely/CHANGELOG.md]: sorry about the typo, updated

**Review by emarsha94 — APPROVED** [review]

**Review by Kyle-Neale — APPROVED** [review]

**Pull request merged**